tests/Operation/WatchFunctionalTest.php

```diff
@@ -269,13 +315,26 @@ public function testNextCannotExtractResumeToken()
         $operation = new Watch($this->manager, $this->getDatabaseName(), $this->getCollectionName(), $pipeline, ['maxAwaitTimeMS' => 100]);
         $changeStream = $operation->execute($this->getPrimaryServer());
 
-        $changeStream->rewind();
```

Comment:
I assume this was removed because `rewind()` now throws its own exception and would prevent us from testing `next()`

I'd suggest replacing with:

```
/* Note: we intentionally do not start iteration with rewind() to ensure
 * that we test extraction functionality within next(). */
```

Borrowed from the socket exception test.

Reply:
Side thought: does this actually throw or does it only introduce a delay with its own `getMore`? Since we've yet to insert a document, I'd expect `rewind()` to not find any result to attempt extraction.

Reply:
You're right, the rewind() doesn't throw because it doesn't attempt extraction.

Reply:
This now makes for very amusing `diff` output.